code/clean/clean_eeoc.do

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+
+Cleans EEOC data 
+
+*******************************************************************************/
+
+import delimited "$raw_data/EEOC/cases_edited.csv", varnames(1) clear
+
+	
+/*******************************************************************************
+Rename Variables
+*******************************************************************************/
+
+ren casename resp_org
+ren civilactionnumber Civil_Action_Number
+ren court court_name
+ren courtfilingdate Court_Filing_Date_temp
+ren resolutiondate Resolution_Date_temp
+ren relief Relief_temp
+ren allegations basis
+
+ren *, lower
```

Comment:
Do we still need to ren *, lower or can we just rename variables directly to lower case in lines 14-20?